integration_tests/sdk/aqueduct_tests/test_metrics/constant/requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
For each of these requirements.txts, can you add a comment that these only need to be added due to Conda? Otherwise it's hard to know since user's typically don't do this.